docs/spl-docs/stats-command.md

```diff
@@ -0,0 +1,96 @@
+# stats Command
+
+## Description
+
+Calculates aggregate statistics, such as average, count, and sum, across the results set. When the `stats` command is utilized without a `BY` clause, a single row is returned, representing the aggregation across the whole set of incoming results. If a `BY` clause is used, one row is generated for each unique value specified in the `BY` clause.
+
+
+## Syntax
+
+**Simple:**  
+stats (stats-function(field) [AS field])... [BY field-list]
+
+**Complete:**  
+Required syntax is in **bold**.
+
+| stats  
+**( \<stats-agg-term\>... )**  
+[\<by-clause\>]  
+
+### Required Arguments
+
+#### stats-agg-term  
+**Syntax:** \<stats-func\>(\<evaled-field\> | \<wc-field\>) [AS \<wc-field\>]\  
+**Description:** A statistical aggregation function. This function can be applied to an eval expression, or to a single field or multiple fields. Utilize the AS clause to assign the result to a new field with a name you choose. You can use wild card characters in field names.
+
```

Comment:
How are fields with wildcards handled? How does `stats` know which field to aggregate then? Or if it's the `AS` field, how does it know what to name it?

Reply:
We do not support using wildcard characters in field names yet, I removed the line `You can use wild card characters in field names`